Core/automation/lib/python/core/utils.py

```diff
@@ -0,0 +1,171 @@
+# -*- coding: utf-8 -*-
+import random
+import time
+
+import configuration
+from core.log import logging, LOG_PREFIX
+from core.jsr223 import scope
+
+from org.joda.time import DateTime
+from org.joda.time.format import DateTimeFormat
+
```

Comment:
> I think it's wrong to put them in the configuration file since they should never be configured. They are more used like constants that never change (With the exeption of Wind_Texts that can be translated if needed)

I think of configuration.py as data storage for keeping personal data private and as a single source for access by multiple modules/scripts. IMO, it doesn't matter if the data is configurable or static, so it seems to me like the right place to put them.

Reply:
> I think of configuration.py as data storage for keeping personal data private and as a single source for access by multiple modules/scripts. IMO, it doesn't matter if the data is configurable or static, so it seems to me like the right place to put them.

In most cases it's our internal data representation that we might want to add to or change in the future. Anyway, I will suggest where to put them and then you will have the opportunity to review them in my coming PR.

Reply:
Another thought on this. These look like they would be seldom used... maybe only from one script. In which case, it may be best to leave them in the script. If multiple rules use them, then just have them all in the same script file.

Reply:
> Another thought on this. These look like they would be seldom used... maybe only from one script. In which case, it may be best to leave them in the script. If multiple rules use them, then just have them all in the same script file.

I agree. I've already relocated most of them. Some are my personal stuff and they will probably end up in my configuration file as you suggested.